x-pack/legacy/plugins/uptime/public/lib/adapters/framework/kibana_global_help.tsx

```diff
@@ -4,23 +4,42 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { EuiLink } from '@elastic/eui';
+import { EuiLink, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n/react';
 import React from 'react';
 
 export const renderUptimeKibanaGlobalHelp = () => (
-  <EuiLink
-    aria-label={i18n.translate('xpack.uptime.header.helpLinkAriaLabel', {
-      defaultMessage: 'Go to our discuss page',
-    })}
-    href="https://discuss.elastic.co/c/uptime"
-    target="_blank"
-  >
-    <FormattedMessage
-      id="xpack.uptime.header.helpLinkText"
-      defaultMessage="Give Uptime feedback"
-      description="The link is to a support form called 'Discuss', where users can submit feedback."
-    />
-  </EuiLink>
+  <EuiFlexGroup direction="column">
+    <EuiFlexItem>
+      <EuiLink
+        aria-label={i18n.translate('xpack.uptime.header.docsLinkAriaLabel', {
+          defaultMessage: 'Go to Uptime documentation',
+        })}
+        href="https://www.elastic.co/guide/en/kibana/current/xpack-uptime.html"
```

Comment:
I'd like to see this link versioned to match the Kibana version being used. Not an exact 1 for 1 example, but we do this in APM like this:

https://github.com/elastic/kibana/blob/0ff97af37f343ea5b087d44d3abf540eec6cd90b/src/legacy/core_plugins/kibana/server/tutorials/apm/index.js#L92

Can you make use of `{config.docs.version}` to version this link? Something like:

`https://www.elastic.co/guide/en/kibana/{config.docs.version}/xpack-uptime.html`?

Reply:
I'll have to investigate, will post back here when I have a chance.

Reply:
Sounds good Justin. I should also mention that I built this as-is and everything works as expected!

Reply:
@bmorelli25 the Kibana team recommended a solution like this, WDYT? https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/canvas/public/lib/documentation_links.js

Reply:
Perfect!

Reply:
@bmorelli25 check out https://github.com/elastic/kibana/pull/40645/commits/d072b95a02f1bbca23b89e4998ebf4ef04e72ec9 and let me know how you're feeling about it.